.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
You could add a name denoting the purpose of this actions like below

 ```
  - name: Checkout code
      uses: actions/checkout@v4

  - name: Setup Node.js
      uses: actions/setup-node@v4
```

Reply:
good suggestions, added some names for each action.